package exception;

public class ExceptionTest38 {

	public static void main(String[] args) {
		

	}

}
